### PR TITLE
[Validator] Added missing translations for Afrikaans

### PR DESCRIPTION
**Pull request by thorne51**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #30148 
| License       | MIT

Added missing translations for Afrikaans

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 1]
```diff
@@ -222,6 +222,118 @@
...
+                <source>This value is not a valid ISBN-10.</source>
+                <target>Hierdie waardie is nie 'n geldige ISBN-10 nie.</target>
+            </trans-unit>
+            <trans-unit id="61">
+                <source>This value is not a valid ISBN-13.</source>
+                <target>Hierdie waarde is nie 'n geldige ISBN-13 nie.</target>
+            </trans-unit>
+            <trans-unit id="62">
+                <source>This value is neither a valid ISBN-10 nor a valid ISBN-13.</source>
+                <target>Hierdie waarde is 'n geldige ISBN-10 of ISBN-13 nie.</target>
```
Comment (pierredup) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Missing `nie` after `waarde is`

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 2]
```diff
@@ -222,6 +222,118 @@
                 <source>Unsupported card type or invalid card number.</source>
                 <target>Nie-ondersteunde tipe kaart of ongeldige kredietkaart nommer.</target>
             </trans-unit>
+            <trans-unit id="59">
+                <source>This is not a valid International Bank Account Number (IBAN).</source>
+                <target>Hierdie is nie 'n geldige Internationale Bank Rekening Nommer (IBAN) nie.</target>
+            </trans-unit>
+            <trans-unit id="60">
+                <source>This value is not a valid ISBN-10.</source>
+                <target>Hierdie waardie is nie 'n geldige ISBN-10 nie.</target>
```
Comment (pierredup) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Extra `i` in `waardie` can be removed

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 3]
```diff
@@ -222,6 +222,118 @@
...
+                <source>The image is square ({{ width }}x{{ height }}px). Square images are not allowed.</source>
+                <target>Die beeld is vierkantig ({{ width }}x{{ height }}px). Vierkantige beelde word nie toegelaat nie.</target>
+            </trans-unit>
+            <trans-unit id="76">
+                <source>The image is landscape oriented ({{ width }}x{{ height }}px). Landscape oriented images are not allowed.</source>
+                <target>Die beeld is landskap georiënteerd ({{ width }}x{{ height }}px). Landskap georiënteerde beelde word nie toegelaat nie.</target>
+            </trans-unit>
+            <trans-unit id="77">
+                <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
+                <target>Die beel dis portret georiënteerd ({{ width }}x{{ height }}px). Portret georiënteerde beelde word nie toegelaat nie.</target>
```
Comment (pierredup) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Missing `d` in `beeld`

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 4]
```diff
@@ -222,6 +222,118 @@
...
+                <source>This value should not be equal to {{ compared_value }}.</source>
+                <target>Hierdie waarde moet nie dieselfde as {{ compared_value }} wees nie.</target>
+            </trans-unit>
+            <trans-unit id="72">
+                <source>This value should not be identical to {{ compared_value_type }} {{ compared_value }}.</source>
+                <target>Hierdie waarde moet nie identies as {{ compared_value_type }} {{ compared_value }} wees nie.</target>
+            </trans-unit>
+            <trans-unit id="73">
+                <source>The image ratio is too big ({{ ratio }}). Allowed maximum ratio is {{ max_ratio }}.</source>
+                <target>Die beeld aspek is te groot ({{ ratio }}). Toegelaat maksimum aspek is {{ max_ratio }}.</target>
```
Comment (pierredup) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: `Toegelaat` can be `Toegelate` (same below). or maybe the last part can be `Die maksimum toegelate aspek is  {{ max_ratio }}.`

**Review by pierredup — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf (hunk 4 above)
Comment (thorne51) [reply src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Opted for `Die maksimum toegelate aspek is {{ max_ratio }}.` as it reads better. Thanks @pierredup

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf (hunk 3 above)
Comment (thorne51) [reply src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Fixed

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf (hunk 2 above)
Comment (thorne51) [reply src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Fixed

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf (hunk 1 above)
Comment (thorne51) [reply src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: Fixed

**Comment by thorne51** [comment]
status: Needs review

**Review by pierredup — APPROVED** [review]
Fabbot failure is a false-positive

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 5]
```diff
@@ -222,6 +222,118 @@
...
+                <source>This is not a valid UUID.</source>
+                <target>Hierdie is nie 'n geldige UUID nie.</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Hierdie waarde moet 'n veelvoud van {{ compared_value }} wees.</target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Hierdie Besigheids Identifikasie Kode (BIK) is nie geassosieer met IBAN {{ iban }} nie.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: We should not translate the `BIC` acronym (like we don't translate the `IBAN` acronym either). Thanks.

**Review by javiereguiluz — APPROVED** [review]
@thorne51 thanks for your contribution! This translation was missing lots of contents, so thanks for the extra effort you made to complete it.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @thorne51.

**Pull request merged**